client/src/components/Months.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const MonthWrapper = styled.section`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  font-family: Circular, -apple-system, BlinkMacSystemFont, Roboto, Helvetica Neue, sans-serif;
+  color: rgb(72, 72, 72) !important;
+  font-size: 18px !important;
+  text-align: center !important;
+  font-weight: bold;
+`;
+
+const PrevMonthButton = styled.button`
+`;
```

Comment:
Not too sure how styled components work, but is this line necessary considering there's no additional styling?